### PR TITLE
Added vctl install agent-dir jira_179

### PR DESCRIPTION
**Pull request by craig8**
# Description

This pull request is adding vctl install agent-dir to replace the scripts/install-agent.py script.  It includes changing the tests to utilize this functionality from the command line.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

This has been tested using the actions as a basis for the newest way of testing

### CODE REVIEW

**Comment by shwethanidd** [comment]
Add the jia number in the description. I can point this as example to Chandrika

File: volttron/platform/install_agents.py [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+            data = yaml.safe_load(fp)
+    except:
+        _log.error("Invalid yaml/json config file.")
+        sys.exit(-10)
+
+    # Configure the whl file before installing.
+    add_files_to_package(opts.package, {'config_file': config_file})
+    env = os.environ.copy()
+
+    if agent_exists:
```
Comment (shwethanidd) [inline volttron/platform/install_agents.py:147]: We should upgrade only if 'force' is provided

**Review by shwethanidd — CHANGES_REQUESTED** [review]
Please move github workflows feature into a different PR. Is there a jira ticket for it?

File: volttron/platform/install_agents.py (hunk 1 above)
Comment (craig8) [reply volttron/platform/install_agents.py:147]: This is actually found up above...this if is for upgrade or install.

**Pull request closed without merging**